CHANGES.md

```diff
@@ -1,3 +1,9 @@
+### Unreleased
+
+#### Changed
+- `sync` has to be called by the read-only instance to synchronise with the
+  files on disk. (#1008)
```

Comment:
```suggestion
- **irmin-pack**
  - `sync` has to be called by the read-only instance to synchronise with the
     files on disk. (#1008)
```

This PR also changes the meaning of `sync` (which is now `flush`). Perhaps need another sentence to explain that.

Reply:
Thanks, added a sentence about flush.